### PR TITLE
TSK-1742: Bump @Angular from 12.0.5 to 12.2.7

### PR DESCRIPTION
**Pull request by Tristan2357**
Also removed deprecated/ignored compiler options

---
 
### For the submitter:
- [ ] I updated the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) and will supply links to the specific files
- [x] I did not update the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview)
- [ ] I included a link to the [SonarCloud branch analysis](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1019969636/SonarCloud+Integration)
- [ ] I added a description of changes on the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [x] I did not update the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [x] I put the ticket in review
- [ ] After integration of the pull request, I verified our [bluemix test environment](http://taskana.mybluemix.net/taskana) is not broken

### Verified by the reviewer:
- [ ] Commit message format → TSK-XXX: Your commit message.
- [ ] Submitter's update to [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) is sufficient
- [ ] SonarCloud analysis meets our standards
- [ ] Update of the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana) reflects changes
- [ ] PR fulfills the ticket
- [ ] Edge cases and unwanted side effects are tested
- [ ] Readability

### CODE REVIEW

File: web/angular.json [hunk 1]
```diff
@@ -57,10 +57,7 @@
         "serve": {
           "builder": "@angular-devkit/build-angular:dev-server",
           "options": {
-            "browserTarget": "taskana-web:build",
-            "sourceMap": {
-              "scripts": true
-            }
```
Comment (mustaphazorgati) [inline web/angular.json:63]: why did you remove the sourceMap?

File: web/src/tsconfig.app.json [hunk 2]
```diff
@@ -6,8 +6,5 @@
     "types": ["jest"]
   },
   "files": ["main.ts", "polyfills.ts"],
-  "include": ["src/**/*.d.ts"],
-  "angularCompilerOptions": {
-    "enableIvy": false
-  }
```
Comment (mustaphazorgati) [inline web/src/tsconfig.app.json:12]: what's this and why did you remove this?

**Review by mustaphazorgati — CHANGES_REQUESTED** [review]

File: web/src/tsconfig.app.json (hunk 2 above)
Comment (Tristan2357) [reply web/src/tsconfig.app.json:12]: Angular switched in version 9 from their ViewEngine to Ivy as their default rendering pipeline.
We used this option to force the use of the ViewEngine, but this option has been removed as of Angular 11.
All this option currently does is produce a warning when starting our frontend.

File: web/angular.json (hunk 1 above)
Comment (Tristan2357) [reply web/angular.json:63]: this usage is deprecated
but a complete removal was unnecessary 
i will readd it
